resources/metadata/sonar.yaml

```diff
@@ -80,6 +117,14 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+      - name: inferBranchName
+        type: bool
+        description: "Jenkins only: Whether to infer the `branchName` parameter automatically based on the
```

Comment:
I would remove the `Jenkins only: ` label as it is not really Jenkins specific.

Reply:
The implementation for this is in the Groovy wrapper. `branchName` is auto-filled from the wrapper only, the go-implementation does not look at this parameter.